### DISCUSSION TITLE
redirection links?

### DISCUSSION THREAD

**Discussion by fangyuan404** [sumatrapdfreader/sumatrapdf#4014] [Q&A]
Does sumatrapdf supports redirection links?
I would like to quickly redirect from other software to a certain location in a sumatrapdf file

**Comment by GitHubRulesOK** [comment]
unsure what you mean by redirection
the command line cannot accept redirected text inputs but uses Dynamic Data Embedding  for control by system applications 

command line can accept -switches such as goto page X or zoom to Z or several other navigation commands and some of the Adobe ones too.
